src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DateValue.txt

```diff
@@ -37,4 +37,13 @@ true
 "2/14/2001 6:01 AM"
 
 >> Second(DateTimeValue("02/14/2001 6:00 AM") - 1.1574e-5)
-59
\ No newline at end of file
+59
+
+>> Text(DateTimeValue("jeudi 21 juillet 2022 19:34:03", "fr-FR"))
```

Comment:
>Text [](http://example.com/codeflow?start=3&length=4)

fyi, I double checked that Power Apps displays the same results here for each case.

Reply:
Thanks!